deploy/stacks/lambda_api.py

```diff
@@ -248,48 +255,60 @@ def __init__(
 
             for claims_map in custom_auth.get('claims_mapping', {}):
                 custom_lambda_env[claims_map] = custom_auth.get('claims_mapping', '').get(claims_map, '')
+        else:
+            custom_lambda_env = {
+                'envname': envname,
+                'LOG_LEVEL': 'DEBUG',
+                'user_info_url': f'https://{user_pool_domain.domain_name}.auth.{self.region}.amazoncognito.com/oauth2/userInfo',
+                'custom_auth_provider': 'Cognito',
+                'custom_auth_url': f'https://cognito-idp.{self.region}.amazonaws.com/{user_pool.user_pool_id}',
```

Comment:
instead of passing the user_info_url explicitly we can use `https://{custom_auth_url}/.well-known/openid-configuration` and read the userinfo_endpoint field.

Reply:
using `https://{custom_auth_url}/.well-known/openid-configuration` for userinfo URL now

Reply:
should we remove the `custom_auth_jwks_url` since you are pulling this from `openid-configuration` endpoint?

Reply:
removed in latest